tests/water_fugacity_prefactor.prm

```diff
@@ -0,0 +1,123 @@
+# A test for the exact water fugacity prefactor
+# with the visco_plastic material model.
+# Following the flow law for wet diffusion from
+# Hirth & Kohlstaedt 2004 the exact viscosity is
+# calculated in python through:
+# import numpy as np
+# m = 3
+# d = 1e-3 m
+# E = 375e3 J/mol
+# V = 15e-6 m^3/mol
+# R = 8.3144621
+# P = 10e9 Pa
+# r = 0.7
+# T = 1673 K
+# A_diff = 2.5e7 / 1e6 / 1e6**m / 1e6**r 1/Pa^2/m^3/s
+
+
+# A_h2o = 26 / 1e6**r 1/Pa
+# E_h2o = 40e3 J/mol
+# V_h2o = 10e-6 m^3/mol
+
+# M_h2o = 18.01528
+# M_ol = 147.0027
+
+# wt_fraction_h2o = 0.01
+# wt_ol = 1 - wt_fraction_h2o
+# COH = (wt_fraction_h2o/M_h2o) / (wt_ol/M_ol) * 1e6
+
+# fh2o = COH / A_h2o * np.exp((E_h2o + P*V_h2o)/(R * T))
+# viscosity_analytic = 0.5 * A_diff**(-1) * d**(m) * np.exp((E + P*V) / R / T) * fh2o**(-r)
+# In the background composition, the viscosity should be equal to 2.766e19,
```

Comment:
Looking farther down in the test setup, the solid_2 field is only present above 50 km depth. This is fine, but I think it would also be helpful to output the visualization files (gnu format) to confirm that the viscosity does change as you cross the boundary from the background to the solid_2 material.